docs/0-installation.md

```diff
@@ -112,6 +112,30 @@ Draper::CollectionDecorator.send :delegate, :per_page_kaminari
 
 If you're getting the error `wrong number of arguments (6 for 4..5)`, [read #2703].
 
+## webpacker
+
+Since rails 6.0 webpacker has become the default asset generator. You can opt in using webpacker for ActiveAdmin assets as well by setting `use_webpacker` at install or manually.
```

Comment:
I'm not a great prose writer, but I added a small suggestion that maybe reads a bit better?

```suggestion
Since rails 6.0, webpacker has become the default asset generator. You can opt into using webpacker for ActiveAdmin assets as well by setting `use_webpacker` at installation time or manually.
```